hoot-core/src/main/cpp/hoot/core/cmd/ChangesetDeriveReplacementCmd.cpp

```diff
@@ -74,15 +74,6 @@ class ChangesetDeriveReplacementCmd : public BoundedCommand
   {
     // process optional params
 
-    // TODO: eventually this check can go away
```

Comment:
Everything remove here is no longer an option and set automatically based on whether the C&R or cut only workflow is in use.